src/sentry/static/sentry/app/components/organizations/multipleProjectSelector.jsx

```diff
@@ -43,90 +40,84 @@ export default class MultipleProjectSelector extends React.Component {
     onUpdate();
   };
 
+  handleClear = () => {
+    this.props.onChange.bind(this, []).call();
```

Comment:
Isn't this the same as `this.props.onChange.call(this, [])`?

Also is it even necessary to bind `this`?
```suggestion
    this.props.onChange([])
```

Reply:
probably not lol. a little bit of a cut-and-paste scenario